docs/design-language.md

```diff
@@ -223,4 +223,33 @@ Panels come with an assortment of configurable navigation controls:
 
 ![panels](/img/panels/panel-welcome.jpg "Welcome Dialog Panel") ![panels](/img/panels/panel-feature.jpg "Features Panel")
 
+## Dialogs
+
+### Modal Dialogs
+
+Occasionally, there will be a need to display information, warn the user, or present controls in a manner that needs to utilize a more focused interface to the user.
+
+![tabs](/img/dialogs/dialog.png "Dialog")
+
+Examples of this include:
```

Comment:
Screenshots would be really helpful with these examples.

Reply:
Done. Examples Added in.